app/src/main/java/com/github/se/icebreakrr/model/profile/Profile.kt

```diff
@@ -24,7 +24,8 @@ data class Profile(
     val description: String,
     val tags: List<String> = listOf(),
     val profilePictureUrl: String? = null,
-    val fcmToken: String? = null
+    val fcmToken: String? = null,
+    var hasBlocked: List<String> = listOf(),
```

Comment:
Ok so, even though that's a good idea, I tried to implement it but I can't seem to make it work.

If I modify it here, then I need to modify the `ProfileRepositoryFirestore` to read it as a Set instead of a List. But when I put `(document.get("hasBlocked") as? Set<*>)` it's recognized as `collection`...
I tried to work it around by reading it as a `List` then apply `.toSet()`, but then the app just crashes.

I will dig into this in M3, but I'll leave it as a List for now.